src/SAML2/Message.php

```diff
@@ -226,10 +228,8 @@ private function validateSignature(\DOMElement $xml)
      * @param mixed    $data     The data that should be included as the first parameter to the function
      * @return void
      */
-    public function addValidator($function, $data)
+    public function addValidator(callable $function, $data)
```

Comment:
Same here, I think callable is not available for type hinting in PHP 5.x.

Reply:
Since 5.4
http://php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration